go/chat/sender.go

```diff
@@ -314,23 +316,52 @@ func (s *Deliverer) deliverLoop() {
 
 type NonblockingSender struct {
 	libkb.Contextified
-	sender Sender
+	sender           Sender
+	ti               keybase1.TlfInterface
+	identifyBehavior keybase1.TLFIdentifyBehavior
 }
 
-func NewNonblockingSender(g *libkb.GlobalContext, sender Sender) *NonblockingSender {
+func NewNonblockingSender(g *libkb.GlobalContext, sender Sender, ti keybase1.TlfInterface, identifyBehavior keybase1.TLFIdentifyBehavior) *NonblockingSender {
 	s := &NonblockingSender{
-		Contextified: libkb.NewContextified(g),
-		sender:       sender,
+		Contextified:     libkb.NewContextified(g),
+		sender:           sender,
+		ti:               ti,
+		identifyBehavior: identifyBehavior,
 	}
 	return s
 }
 
-func (s *NonblockingSender) Prepare(ctx context.Context, plaintext chat1.MessagePlaintext, convID *chat1.ConversationID) (*chat1.MessageBoxed, error) {
+func (s *NonblockingSender) Prepare(ctx context.Context, plaintext chat1.MessagePlaintext, convID *chat1.ConversationID) (*chat1.MessageBoxed, []keybase1.TLFUserBreak, error) {
 	return s.sender.Prepare(ctx, plaintext, convID)
 }
 
 func (s *NonblockingSender) Send(ctx context.Context, convID chat1.ConversationID,
-	msg chat1.MessagePlaintext) (chat1.OutboxID, *chat1.RateLimit, error) {
+	msg chat1.MessagePlaintext) (chat1.OutboxID, *chat1.RateLimit, []keybase1.TLFUserBreak, error) {
+	// The deliverer uses GUI mode all the time, i.e., no errors on identify
```

Comment:
I guess so. Is this useful only to GUI then? If so, I'd feel safer to have it renamed to `GUINonblockingSender`, in case we accidentally use it for CLI (or other things) in the future. What do you think?